libsolidity/interface/CompilerStack.cpp

```diff
@@ -1080,7 +1102,7 @@ string CompilerStack::createMetadata(Contract const& _contract) const
 {
 	Json::Value meta;
 	meta["version"] = 1;
-	meta["language"] = "Solidity";
+	meta["language"] = m_importedSources ? "SolidityAST" : "Solidity";
```

Comment:
Any particular reason for this?

Reply:
I thought that as the metadata contains only the hashed source, it would not be obvious that it originates from an AST, so it would make sense to state that there.